### PR TITLE
update to 6.1 kernel

### PR DESCRIPTION
**Pull request by wiktorek140**
Its at lease compiling. Not tested. Will be nice if revieved by someone with more experience in that matter.

Signed-off-by: Wiktor Strzębała <wiktorek140@gmail.com>

### CODE REVIEW

**Comment by wiktorek140** [comment]
I can confirm, that after building, it works, wifi connects properly

File: os_dep/linux/os_intfs.c [hunk 1]
```diff
@@ -1680,7 +1680,14 @@ int rtw_os_ndev_register(_adapter *adapter, const char *name)
 	u8 rtnl_lock_needed = rtw_rtnl_lock_needed(dvobj);
 
 #ifdef CONFIG_RTW_NAPI
-	netif_napi_add(ndev, &adapter->napi, rtw_recv_napi_poll, RTL_NAPI_WEIGHT);
+	netif_napi_add(
+		ndev,
+		&adapter->napi,
+		rtw_recv_napi_poll
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 0))
+		RTL_NAPI_WEIGHT
```
Comment (arthurbambou) [inline os_dep/linux/os_intfs.c:1688]: Missing comma?

**Comment by revant** [comment]
This worked for me.

https://aur.archlinux.org/packages/rtl8821ce-dkms-git#comment-894933

**Comment by TheCourierNV** [comment]
I also tested this on Arch and it works.

**Comment by CaneMandarinesco** [comment]
This works! compiles, and wifi never falls out.

**Comment by uroborosq** [comment]
Have been using for 4 days, no issues noticed :+1:

**Comment by verchyle** [comment]
Also I've been using it for a couple of days now on arch, no issues, everything works fine.

**Comment by tomaspinho** [comment]
Thank you for this. Merging.

**Pull request merged**